runtime/queries/svelte/injections.scm

```diff
@@ -0,0 +1,26 @@
+; injections.scm
+; --------------
```

Comment:
Injections should follow https://tree-sitter.github.io/tree-sitter/syntax-highlighting#language-injection